### DISCUSSION TITLE
https://plugin-blog2.vuejs.press/demo.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#491] [Announcements]
# https://plugin-blog2.vuejs.press/demo.html

We provide a demo using vuepress-plugin-blog2 and @vuepress/theme-default for you to better understand how to use the plugin. Source Code (https://github.com/vuepress-theme-hope...

https://plugin-blog2.vuejs.press/demo.html

**Comment by wired247** [comment]
## This is a markdown comment
What kind of formatting can I use?
New paragraph, some *italic* and **bold**
